python/desc/imsim/camera_readout.py

```diff
@@ -467,11 +469,11 @@ def write_fits_file(self, outfile, overwrite=True, run_number=None,
             run_number = self.visit
         output[0].header['RUNNUM'] = str(run_number)
         output[0].header['DARKTIME'] = output[0].header['EXPTIME']
-        mjd_obs = astropy.time.Time(output[0].header['MJD-OBS'], format='mjd')
-        output[0].header['DATE-OBS'] = mjd_obs.isot
+        output[0].header['TIMESYS'] = 'TAI'
         output[0].header['LSST_NUM'] = lsst_num
         output[0].header['TESTTYPE'] = 'IMSIM'
         output[0].header['IMGTYPE'] = 'SKYEXP'
+        output[0].header['OBSTYPE'] = 'SKYEXP'
```

Comment:
Do we want to make this one always mirror IMGTYPE? Or allow them to diverge?

Reply:
I think they will always be the same.  I didn't want to just summarily get rid of `IMGTYPE` since I think there is code that still looks for that keyword to be set, e.g., when making calibration products.  The scripts that generate those files will need to add `OBSTYPE`.

Reply:
I was  wondering if (as long as it still exists) to be defensive if we should do 

output[0].header['OBSTYPE'] = output[0].header['IMGTYPE']

to stop someone from accidentally just changing only one in the future.

Reply:
If we think OBSTYPE is going to be around longer shouldn't we set IMGTYPE from OBSTYPE?